bbc/simorgh
# Duplicate key prop

Resolves #2061

**Overall change:** Fix for duplicate keys in components by making use of the `uuid` package.

**Code changes:** 
- Added uuid keys to the caption component.

**Testing notes:**
- Go to the storybook page (http://localhost.bbc.com:9001/?path=/story/figure--with-a-caption) and there should not be an error about duplicate keys.
 - Update an article that has multiple paragraphs in a caption: make them have the same text in the multiple paragraphs - see no error about duplicate keys.

---
